source/styleEditor.js

```diff
@@ -1,3 +1,15 @@
+
 $(document).ready(function(){
-  // your code goes here.
+
+  $('#style_editor').submit(function(){
+    var selector = $('input[name=selector]').val();
+    var property = $('input[name=property]').val();
+    var value = $('input[name=value]').val();
+    $(selector).css(property, value);
+  });
+
+  $('form').on('submit', function(event){
+    event.preventDefault()
```

Comment:
i suggest just putting this at the top of your first submit event handler (between lines 4 and 5). no need to bind another event just to prevent the default.